templates/order/details.html

```diff
@@ -61,6 +62,39 @@ <h2 class="order-details__header">
         {% endif %}
       {% endif %}
       {% include "order/_ordered_items_table.html" %}
+      {% if order.status == 'open' %}
```

Comment:
There are two problems here:
1. A hardcoded value of the order status. Instead, we should have the property `order.is_open` defined in the model.
2. This hides all notes if an order is closed. Only the note form should be hidden in such case.